feature-flags.json

```diff
@@ -5,6 +5,9 @@
     },
     "pageDashboard": {
       "enabled": "development"
+    },
+    "adminBar": {
+      "enabled": "development"
```

Comment:
@asvinb is it widget related feature? If not, it should be declared outside of the widgets scope.

Reply:
@eugene-manuilov It's not a widget related feature, however as per the AC:
_While this won't for now use the actual Widgets API, it still underlies several similar requirements._